app/kubernetes/models/application/models/index.js

```diff
@@ -44,6 +44,29 @@ export class KubernetesApplication {
   }
 }
 
+/**
+ * HelmApplication Model (Composite)
+ */
+const _HelmApplication = Object.freeze({
```

Comment:
please don't use this pattern.

just use a regular class

Reply:
was copying what was above (in file) for consistency.

Will use class though (as it makes more sense) 👍